CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+cmake_minimum_required(VERSION 3.10)
+
+#project_name
+project(UniversityTimetable)
+
+#Specify the C++ standard
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ggdb")
```

Comment:
Why do we always compile with the ggdb flag, I think it is unnecessary (or you can give it by flag)

Reply:
I add it for debugging

Reply:
Yes, but we don't need to build the project with debug, change it as optional (see CMAKE_BUILD_TYPE)

Reply:
okay I removed it